scripts/setup_env

```diff
@@ -58,33 +58,34 @@ esac
 # Kill all sudo in the current shell
 sudo -k
 
-function run_pip {
-    "${PIP}" install \
-             -r "${PROJECT_DIR}ansible-requirements.txt" \
-             -r "${PROJECT_DIR}requirements.txt" \
-             -r "${PROJECT_DIR}test-requirements.txt" \
-             -r "${PROJECT_DIR}molecule-requirements.txt" ${@:-}
-}
-
 # Ensure the required ci file is presnet
 mkdir -p ${HOME}/ci/yum.repos.d
 cp /etc/ci/mirror_info.sh ${HOME}/ci || touch ${HOME}/ci/mirror_info.sh
 cp -r /opt/yum.repos.d/* ${HOME}/ci/yum.repos.d || cp -r /etc/yum.repos.d/* ${HOME}/ci/yum.repos.d
 
-if [ ${USE_VENV} == 'yes' ]; then
-    # Create a virtual env
-    "${PYTHON_EXEC}" -m venv --upgrade-deps "${HOME}/test-python"
-    if [[ -d "${HOME}/.cache/pip/wheels" ]]; then
-        rm -rf "${HOME}/.cache/pip/wheels"
-    fi
-fi
+
+case ${USE_VENV} in
+    y|yes|true):
+        # Create a virtual env
+        "${PYTHON_EXEC}" -m venv --upgrade-deps "${HOME}/test-python"
+        if [[ -d "${HOME}/.cache/pip/wheels" ]]; then
+            rm -rf "${HOME}/.cache/pip/wheels"
+        fi
+        ;;
+esac
 
 # Run bindep
 "${PIP}" install pip setuptools bindep --upgrade
 "${PROJECT_DIR}/scripts/bindep-install"
 
-# Install local requirements
-run_pip
+# Install requirements
+PIP_INSTALL_ARGS=""
+if [ "$EUID" -ne 0 ]; then
+    PIP_INSTALL_ARGS="${PIP_INSTALL_ARGS} --user"
+fi
```

Comment:
even with root user you can install packages with `--user` (so in `${HOME}/.local`), I'd keep this as default behavior while installing packages from pip, to avoid that those packages will be installed in /usr tampering the distro.